sdk/playwrighttesting/microsoft-playwright-testing/src/common/constants.ts

```diff
@@ -105,31 +105,47 @@ export class Constants {
       400: "Bad Request",
       401: "Invalid authentication to run test.",
       403: "You do not have required permissions to run test.",
-      500: "Unexpected server-side issue occurred."
+      500: "The test run service is currently unavailable. Please check the service status and try again.",
```

Comment:
This error message is more suited for 503.
For 500 we could have something like:
"An unexpected error occurred on our server. Our team is working to resolve the issue. Please try again later, or contact support if the problem continues."